execute/entities.go

```diff
@@ -48,3 +51,17 @@ const (
 	// Skipped indicates a client' TestCase did not run
 	Skipped
 )
+
+var StatusNames = map[Status]string{
+	Success: "success",
+	Failed:  "failed",
+	Skipped: "skipped",
+}
```

Comment:
I think what you want to do here instead is,

```go
func (s Status) String() string {
  switch s {
    case Success: return "success"
    // ..
    default: return fmt.Sprintf("Status(%v)", int(s))
}
```

and use that in `MarshalText()`